pkg/storage/plan_general_characteristicsStore.go

```diff
@@ -76,24 +76,23 @@ func (s *Store) PlanGeneralCharacteristicsGetByID(logger *zap.Logger, id uuid.UU
 	return &gc, nil
 }
 
-// PlanGeneralCharacteristicsGetByModelPlanID returns the plan general characteristics for a given model plan id
-func (s *Store) PlanGeneralCharacteristicsGetByModelPlanID(logger *zap.Logger, modelPlanID uuid.UUID) (*models.PlanGeneralCharacteristics, error) {
-	gc := models.PlanGeneralCharacteristics{}
+// PlanGeneralCharacteristicsGetByModelPlanIDLOADER returns the plan GeneralCharacteristics for a slice of model plan ids
+func (s *Store) PlanGeneralCharacteristicsGetByModelPlanIDLOADER(logger *zap.Logger, paramTableJSON string) ([]*models.PlanGeneralCharacteristics, error) {
+	genCharSlice := []*models.PlanGeneralCharacteristics{}
 
-	statement, err := s.db.PrepareNamed(planGeneralCharacteristicsGetByModelPlanIDSQL)
+	stmt, err := s.db.PrepareNamed(planGeneralCharacteristicsGetByModelPlanIDLoaderSQL)
 	if err != nil {
 		return nil, err
 	}
-
 	arg := map[string]interface{}{
```

Comment:
NIT: If you prefer, there is a utility method to help make these string-map interfaces clearer:
CreateNamedIDQueryMap